src/xdocs/config_system_properties.xml

```diff
@@ -16,39 +16,14 @@
       </macro>
     </section>
 
-    <section name="Caching Support">
+    <section name="Localization Support">
       <p>
-        The property <code>checkstyle.cache.file</code>
-        specifies the name of a file that can be used to cache details
-        of files that pass Checkstyle. This can <i>significantly</i>
-        increase the speed of checkstyle on successive runs. The
-        property type is <a href="property_types.html#String">String</a>
-        and defaults to an empty string (which means no caching is done).
-      </p>
-    </section>
-
-    <section name="Localisation Support">
-      <p>
-        Checkstyle supports a mechanism for localising the output
-        messages. If your language is not supported, please consider
+        Checkstyle supports localization of the output messages.
+        Locale is taken from default java property <code>user.language</code>.
+        If your language is not supported, please consider
         translating the messages in the <code>messages.properties</code> file. Please let us
         know if you translate the file.
       </p>
-
-      <p>
-        The property <code>checkstyle.locale.language</code> specifies the
-        language to use in localising the output messages. The property
-        type is <a href="property_types.html#String">String</a> and
-        defaults to the default system locale language.
-      </p>
-
-      <p>
-        The property <code>checkstyle.locale.country</code>
-        specifies the country to use in localising the output
-        messages. The property type is <a
-        href="property_types.html#String">String</a> and defaults to the
-        default system locale country.
-      </p>
     </section>
 
     <section name="Base directory support">
```

Comment:
How come this is staying? It also is a Checker property and not a recognized property. https://checkstyle.org/config.html#Checker_Properties

Reply:
:) good catch, it is left over from Checkstyle as ANT task.

Reply:
I removed it again